### ISSUE TITLE
opamps are too perfect

### ISSUE THREAD

**Issue by Bushmills** [Proto-App/Proto-Android#12]
If output is fed back to non-inverting input, you'd expect it to eventually drop out of equilibrium. Adding a tiny bit of noise could achieve this.

**Comment by Proto-App** [comment]
Yes, current opamp is ideal. We will bring more real component in feature (maybe even one of well recognized ICs)
